### PR TITLE
DS-2142 Road Distance Performance Profile

### PR DESCRIPTION
**Pull request by kayodewale**

### CODE REVIEW

**Review by DomBaker — DISMISSED** [review]

**Review by swestbro1 — DISMISSED** [review]
LGTM

File: infrastructure/stacks/api-stage/main.tf
Comment (swestbro1) [inline infrastructure/stacks/api-stage/main.tf:1]: Do we want to keep in the vented log work around for the merge or have them commented as with the invoke part?

**Review by swestbro1 — APPROVED** [review]
LGTM

1 comment about the vented logs

File: infrastructure/stacks/api-stage/main.tf
Comment (swestbro1) [reply infrastructure/stacks/api-stage/main.tf:1]: Ah I see that's why that issue popped up

**Review by DomBaker — APPROVED** [review]

**Review by DomBaker — APPROVED** [review]

**Pull request merged**